Tests/Common/IntegrationTestFixtureBase.Asserts.cs

```diff
@@ -350,21 +350,21 @@ private async Task<SearchLogResult> SearchTcpLogs(Func<SearchLogEvent, bool> pre
             return new SearchLogResult(false, processedEvents);
         }
 
-        private async Task<SearchLogResult> SearchTcpLogs(Func<string, bool> predicate, SearchLogOptions options = null)
+        private async Task<SearchLogResult> SearchTcpLogs(Func<string, bool> predicate, SearchLogOptions options)
         {
             return await SearchTcpLogs(evt => predicate(evt.Message), options);
         }
 
-        private async Task<SearchLogResult> SearchIoTHubLogs(Func<SearchLogEvent, bool> predicate, SearchLogOptions options = null)
+        private async Task<SearchLogResult> SearchIoTHubLogs(Func<SearchLogEvent, bool> predicate, SearchLogOptions options)
         {
-            var maxAttempts = options?.MaxAttempts ?? Configuration.EnsureHasEventMaximumTries;
+            var maxAttempts = options.MaxAttempts ?? Configuration.EnsureHasEventMaximumTries;
             var processedEvents = new HashSet<SearchLogEvent>();
             for (var i = 0; i < maxAttempts; i++)
             {
                 if (i > 0)
                 {
                     var timeToWait = i * Configuration.EnsureHasEventDelayBetweenReadsInSeconds;
-                    if (!string.IsNullOrEmpty(options?.Description))
+                    if (!string.IsNullOrEmpty(options.Description))
```

Comment:
[nit] this is a statement that should now never be true, no (there are other occurrences, just commenting this once)?

Reply:
After the changes in fc631626d0e6c7a52dd6aea24183a4d727d99062 and 2a4c194d98605f899b707c9092af81c5535acce7 that is the case yes, so the checks were removed.